.travis.yml

```diff
@@ -1,36 +1,71 @@
 sudo: false
 language: php
 
-php:
-  - 5.6
-  - 7.0
-  - 7.1
+cache:
+  directories:
+    - $HOME/.composer/cache
 
-env:
-  - DRIVER_VERSION=stable
+services:
+  - mongodb
 
 matrix:
+  fast_finish: true 
   include:
     - php: 5.6
-      env: DRIVER_VERSION=stable LEGACY_DRIVER_VERSION=stable
-    - php: 5.6
-      env: DRIVER_VERSION=stable COMPOSER_FLAGS="--prefer-dist --prefer-lowest"
+      env: DRIVER_VERSION=stable COMPOSER_FLAGS="--prefer-dist --prefer-lowest" SERVER_VERSION=3.0
+      addons:
+          apt:
+            sources: 
+                - sourceline: "deb [arch=amd64] http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.0 multiverse"
+                  key_url: "https://www.mongodb.org/static/pgp/server-3.0.asc"
+                - "mongodb-upstart"
+            packages: ['mongodb-org-server']
 
-addons:
-  apt:
-    sources:
-    - mongodb-3.2-precise
-    packages:
-    - mongodb-org-server
+    - php: 5.6
+      env: DRIVER_VERSION=stable SERVER_VERSION=3.4
+      addons:
+          apt:
+            sources: 
+                - sourceline: "deb [arch=amd64] http://repo.mongodb.org/apt/ubuntu precise/mongodb-org/3.4 multiverse"
```

Comment:
Not according to MongoDB official guidelines: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#create-a-list-file-for-mongodb